docs/styles/custom-styles.css

```
@@ -0,0 +1,111 @@
+    --color-scrollbar-track: #F5F5F4;
+    --blockquote-background: rgba(28, 25, 23, 0.05);
+
+    /* Dark-mode variants (read by Dokka JS when toggling to dark) */
+    --color-background-page-dt: #0C0A09;
+    --color-background-nav-dt: #1C1917;
+    --color-background-footer-dt: #1C1917;
+    --color-text-dt: #E7E5E4;
+    --color-text-light-dt: #A8A29E;
+
+    /* Typography */
+    --default-font-family: 'Space Grotesk', system-ui, -apple-system, BlinkMacSystemFont,
```

Stylelint (17.4.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)